src/Traits/MailsMockTrait.php

```diff
@@ -63,6 +63,10 @@ protected function assertMailEquals(string $mailableClass, $emailChain, bool $ex
         $emailChain = $this->prepareEmailChain($emailChain);
         $index = 0;
 
+        if (!$exportMode) {
```

Comment:
Need to move `globalMode` checking to the `MailsMockTrait.assertFixture` method.

```
$globalExportMode = $this->globalExportMode ?? false;

if ($exportMode || $globalExportMode) {
    $this->exportContent($mailContent, $expectedMailData['fixture']);
}
```